convert2rhel/breadcrumbs.py

```diff
@@ -224,6 +224,16 @@ def print_data_collection(self):
             "- Convert2RHEL related environment variables\n\n",
             RHSM_CUSTOM_FACTS_FILE,
         )
+        utils.ask_to_continue()
+
+    def confirm_user_backup(self):
+        # Add in the warning about the system back up here!
+        loggerinst.warnign(
+            "Convert2RHEL modifies the systems during the analysis and then rolls back these "
+            "changes when the analysis is complete. In rare cases, this rollback can fail. "
+            "By continuing, you confirm that you have made a system backup and verified that "
+            "you can restore from the backup."
+        )
```

Comment:
This warning does not need to be inside breadcrumbs, as it is not related to breadcrumbs itself. You could move this function to main, as it would be best used there.

Reply:
Ok, Yeah I know at first I was thinking to put it in main but for whatever reason I just put it in breadcrumbs. I'll go ahead and get it moved to main though!